src/Core/Billing/Extensions/CurrencyExtensions.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Bit.Core.Billing.Extensions;
+
+public static class CurrencyExtensions
+{
+    /// <summary>
+    /// Converts a currency amount in major units to minor units.
+    /// </summary>
+    /// <example>123.99 USD returns 12399 in minor units.</example>
+    public static long ToMinor(this decimal amount)
```

Comment:
Let's add unit tests for these static methods. It should be straightforward since they're pure functions.